inlong-dashboard/src/ui/components/EditableTable/index.tsx

```diff
@@ -69,6 +69,10 @@ export interface EditableTableProps
   // Can add a new line? Default: true.
   canAdd?: boolean;
   canBatchAdd?: boolean;
```

Comment:
`EditableTable` as a base component, I don't think `upsetByFieldKey`, `fieldNameKey`, ... such parameters with obvious meanings. The parameters of `EditableTable` should be abstracted, which is the common capability of the underlying component, and the parameters added above should be implemented in the upper component.

For example, this PR adds a new capability to EditableTable, we can define it as an attribute related to append or overwrite, and the impact on fieldName and fieldKey should be implemented in the upper-level business component of Sink.

Reply:
A key point here is that the general abstraction of the parameters of the underlying components and the concretization of the parameters of the upper components.